code/compute_stats.py

```diff
@@ -0,0 +1,102 @@
+"""
+DeepSlide
+Computes the image statistics for normalization.
+
+Authors: Naofumi Tomita
```

Comment:
Did naofumi author this or did you?

Reply:
This was done by Naofumi using code from the PyTorch forums.  I added it in so we didn't have to rely on hard-coding image means and standard deviations each time we run.

Reply:
LGTM

Reply:
Shouldn't this be calculated on the patch-level, not the WSI level? Because WSI is mostly whitespace.

Reply:
@ntomita @jasonwei20 That's just unclearly explained.  I don't to run it at the patch level due to the overlap, but the idea is that the WSI are actually crops or tissues rather than the full WSI.

Reply:
> The idea is that the WSI are actually crops or tissues rather than the full WSI.
How is the reader supposed to know this? I only ask because this is one of the cases where neural nets "fail silently", the performance will be not too great but it still "works" and so often the user doesn't know why the performance isn't better.

Reply:
@jasonwei20 I added in a note at the top of the README.  I'll commit the fix.